src/pages/subreddit/Subreddit.js

```diff
@@ -0,0 +1,75 @@
+import React, {useEffect, useState} from 'react';
+import {Link, useParams} from "react-router-dom";
+import axios from "axios";
+import './Subreddit.css'
+import Header from "../../components/Header";
+
+function SubredditPage() {
+
+
+    const [redditDetails, setRedditDetails] = useState({});
```

Comment:
`redditDetails` wordt uiteindelijk een object. Een default waarde van `undefined` past hier beter bij
```suggestion
    const [redditDetails, setRedditDetails] = useState();
```